doc/template.html

```diff
@@ -67,5 +67,36 @@ <h3 class="date">$date$</h3>
 $for(include-after)$
 $include-after$
 $endfor$
+<script type="text/javascript">
+  (function() {
+    // write out a basic 'dropdown_link' reference element
+    var dropdown_link = document.createElement('a');
```

Comment:
why an `a` link if it doesn't have a `href`? Should it just be a `button` instead?